pkg/server/apis/v1alpha1/descriptors/integration.go

```diff
@@ -117,4 +117,89 @@ var integration = []definition.Descriptor{
 			},
 		},
 	},
+	{
+		Path:        "/integrations/{integration}/scmrepos",
```

Comment:
`repos`, `branches`, etc are not properties of general `integrations`, I don't think it's suitable to treat `repos`, `branches`, etc as sub-resources of integration.

I'd like to bundle apis for CICD together, such that:
```
/cicd/scms/{scm}/repos
/cicd/scms/{scm}/branches
/cicd/scms/{scm}/tags
```

Reply:
SCM is not limited for CICD. `repos` and `branches` can be regarded as the sub-resources of integrated SCM, although these `resources` are not defined and stored by Cyclone.